.github/workflows/nightly_looped_sanitizers.yml

```diff
@@ -0,0 +1,80 @@
+# Check code with looped compilers' sanitizers
+# This build lasts 6 hours.
+name: Looped_Sanitizers
```

Comment:
I believe we rather use naming like `LoopedSanitizers` or `Looped Sanitizers`... or maybe even change this to `Sanitizers - Looped`

Reply:
Done